src/jobs/test-package.yml

```diff
@@ -10,17 +10,24 @@ parameters:
   node_version:
     description: version of node to run tests against
     type: string
-    default: latest
+    default: '15.6.0'
   upload-coverage:
     description: if true, the job will submit code coverage report to codecov.io
     default: false
     type: boolean
+  os:
+    description: operating system you want to test against
+    type: enum
+    enum: ['linux','win/default']
```

Comment:
I'd prefer to use `windows` as the enum for windows. Is it possible to put the `win/default` image in it's own `windows` executor file? Like what you've done with linux?

Reply:
I needed to copy the orb from windows, made the same, cause it wasnt a good way of calling the orb instead.